src/main/java/serialization/Person.java

```diff
@@ -0,0 +1,25 @@
+package main.java.serialization;
+
+import java.io.Serializable;
+
+public class Person implements Serializable {
+    private int id;
+    private String name;
+
+    public Person(int id, String name) {
+        this.id = id;
+        this.name = name;
+    }
+
+    public int getId() {
+        return id;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public String toString() {
```

Comment:
Try to use generated `toString` method.
Press `ALT+Insert` shortcut and generate it